### PR TITLE
calcurse: new package at 4.6.0

### PR DESCRIPTION
**Pull request by mmatongo**
---

## Description of package

A text-based calendar and scheduling application
## Installed manifest of package

```
/var/db/kiss/installed/calcurse/version
/var/db/kiss/installed/calcurse/sources
/var/db/kiss/installed/calcurse/manifest
/var/db/kiss/installed/calcurse/depends
/var/db/kiss/installed/calcurse/checksums
/var/db/kiss/installed/calcurse/build
/var/db/kiss/installed/calcurse/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/bin/calcurse-vdir
/usr/bin/calcurse-upgrade
/usr/bin/calcurse-caldav
/usr/bin/calcurse
/usr/bin/
/usr/
```

## New package

- [Y] Latest upstream version.
- [Y] I agree to maintain this package.
- [Y] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/calcurse/build [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+sed -i test/Makefile.in -e '/ical-009.sh \\/d'
```
Comment (dylanaraps) [inline community/calcurse/build:3]: We typically prefer this syntax: `sed -i '/ical-009.sh \\/d' test/Makefile.in`

File: community/calcurse/build [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+./configure \
+    --disable-docs \
+    --without-asciidoc \
+    --prefix=/usr \
+    --mandir=/usr/share/man \
+    --sysconfdir=/etc \
```
Comment (dylanaraps) [inline community/calcurse/build:10]: We typically put the directories above the options.

Comment (dylanaraps) [reply community/calcurse/build:10]:
```sh
./configure \
    --prefix=/usr \
    etc \
    etc
```

Comment (mmatongo) [reply community/calcurse/build:10]: I may have to read the style guide indepth this time. 
Thank you again @dylanaraps

File: community/calcurse/build (hunk 1 above)
Comment (mmatongo) [reply community/calcurse/build:3]: Noted.

**Comment by dylanaraps** [comment]
:+1:

**Pull request merged**